src/applications/disability-benefits/526EZ/helpers.jsx

```diff
@@ -120,7 +120,12 @@ export function addPhoneEmailToCard(formData) {
 }
 
 export function prefillTransformer(pages, formData, metadata) {
-  const withDisabilityActionType = set('disabilities', transformDisabilities(formData.disabilities), formData);
+  const { disabilities } = formData;
+  if (!disabilities || !Array.isArray(disabilities)) {
```

Comment:
Should we check for array length is zero if the default parameter is an array/if there will be no disabilities in the array if `no-rated-disabilities-found`?